lib/game.rb

```diff
@@ -0,0 +1,79 @@
+class Game 
+
+WIN_COMBINATIONS =[
+    [0,1,2], 
+    [3,4,5],
+    [6,7,8],
+    [0,4,8],
+    [2,4,6],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8]
+  ]
+
+
+
+attr_accessor :board, :player_1, :player_2
+
+ def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new )
+   @board = board
+   @player_1 = player_1
+   @player_2 = player_2
+   
+  end 
+  
+  def current_player
+    if @board.turn_count % 2 == 0
+      return @player_1
+    else
+      return @player_2
+    end
+  end
+  
+  def won?
+    WIN_COMBINATIONS.each do |i|
```

Comment:
Here you can use [`Enumerable#detect`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-detect) to return the winning combination or `nil` if none is found.

```ruby
  def won?
    WIN_COMBINATIONS.detect do |combo|
      @board.cells[combo[0]] == @board.cells[combo[1]] &&
      @board.cells[combo[0]] == @board.cells[combo[2]] &&
      @board.taken?(combo[0] + 1)
    end
  end
```